hazelcast/src/main/java/com/hazelcast/logging/Log4jFactory.java

```diff
@@ -36,16 +36,19 @@ protected ILogger createLogger(String name) {
 
         public Log4jLogger(Logger logger) {
             this.logger = logger;
-            if (logger.getLevel() == org.apache.log4j.Level.DEBUG) {
-                level = Level.FINEST;
-            } else if (logger.getLevel() == org.apache.log4j.Level.INFO) {
-                level = Level.INFO;
-            } else if (logger.getLevel() == org.apache.log4j.Level.WARN) {
-                level = Level.WARNING;
-            } else if (logger.getLevel() == org.apache.log4j.Level.FATAL) {
-                level = Level.SEVERE;
+            org.apache.log4j.Level log4jLevel = logger.getLevel();
+            if (log4jLevel == org.apache.log4j.Level.DEBUG) {
+                this.level = Level.FINEST;
+            } else if (log4jLevel == org.apache.log4j.Level.INFO) {
+                this.level = Level.INFO;
+            } else if (log4jLevel == org.apache.log4j.Level.WARN) {
+                this.level = Level.WARNING;
+            } else if (log4jLevel == org.apache.log4j.Level.FATAL) {
+                this.level = Level.SEVERE;
+            } else if (log4jLevel == org.apache.log4j.Level.OFF) {
+                this.level = Level.OFF;
             } else {
-                level = Level.INFO;
+                this.level = Level.INFO;
             }
```

Comment:
Here the default level is INFO, where as in the inverse getLevel() method it's OFF.
I know that there's no OFF level there, but shouldn't it be analogous?